docker/official/lib/entry.sh

```diff
@@ -49,6 +49,13 @@ if [[ ! -z "${RUNDECK_ENVARS_UNSETS:-}" ]] ; then
     unset RUNDECK_ENVARS_UNSETS
 fi
 
+# Support Arbitrary User IDs on OpenShift
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "rundeck:x:$(id -u):0:rundeck user:/home/rundeck:/bin/bash" >> /etc/passwd
```

Comment:
I believe the Rundeck user may already exist in `passwd`. It may be more appropriate to replace the existing line/entry to match the runtime uid?

Reply:
yes, that's true. Done with commit c0d4be7:
`sed -i "/rundeck/c\rundeck:x:$(id -u):0:rundeck user:\/home\/rundeck:\/bin\/bash" /etc/passwd`

Reply:
With the sed statement Rundeck won't start anymore because the entry script failed with:
sed: couldn't open temporary file /etc/sedF47F9T: Permission denied

I guess sed will generate this as temporary file, but there are no write permissions in /etc. 

Solution would be to use the echo statement as it was at the beginning